graspologic/plot/plot.py

```diff
@@ -1267,6 +1315,8 @@ def _unique_like(vals):
 # assume that the graph has already been plotted in sorted form
 def _plot_groups(ax, graph, inner_labels, outer_labels=None, fontsize=30):
     inner_labels = np.array(inner_labels)
+    if outer_labels is not None:
+        outer_labels = np.array(outer_labels)
```

Comment:
there was some other code that did this, but I got an error when I input `outer_labels` as a list